### PR TITLE
Receptor: Assign templates jobs

### PR DESCRIPTION
**Pull request by Spredzy**
Depends-On: https://github.com/ansible/project-config/pull/357
Depends-On: https://github.com/project-receptor/receptor/pull/141

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -498,3 +498,12 @@
...
       - system-required
+
+- project:
+    name: github.com/project-receptor/receptor
+    default-branch: devel
+    templates:
+      - system-required
+      - ansible-python-jobs
+      - ansible-python3-jobs
+      - publish-to-pypi
```
Comment (pabelanger) [inline zuul.d/projects.yaml]: If you are not gating, these project-templates are not going to work. As they have gate jobs, so for this, we need to have a discussion

Comment (Spredzy) [reply zuul.d/projects.yaml]: Yeah you are right. I missed that. Let me update this for now and add them as the following for now.

```
 third-party-check:
      jobs:
        - ansible-tox-linters
        - ansible-tox-py36
        - ansible-tox-py37
```

Comment (Spredzy) [reply zuul.d/projects.yaml]: done

**Comment by Spredzy** [comment]
recheck

**Comment by Spredzy** [comment]
recheck

**Pull request merged**